docs/animation.md

```diff
@@ -1,3 +1,5 @@
+> Because `react-vis` has no active owner, it is deprecated in favor of a new charting library that we'll introduce in 2020. This means, that `react-vis` won't receive any patches or new features. We will keep you folks updated here on the new library!
```

Comment:
Perhaps something like:
```
This library is deprecated. Please see `DEPRECATED.md`.
```

So that subsequently we only need to update the DEPRECATED.md and README.md ?